tests/e2e-tests/fixtures/fixture-loaders.js

```diff
@@ -103,13 +105,14 @@ const deleteCoupons = ( ids ) =>
 /**
  * Create Products and call createReviews.
  *
+ * currently this only creates a single product for the sake of reviews.
+ *
+ * @todo  add more products to e2e fixtures data.
```

Comment:
moving this todo created a new todo.

Reply:
it created a comment, but it shouldn't create a new issue on merge (because the title will match) _or_ reopen the existing issue if it was closed. At least that's how it _should_ work (we'll see!)